coordinator/src/main/scala/com/socrata/datacoordinator/service/Service.scala

```diff
@@ -221,7 +224,32 @@ class Service(serviceConfig: ServiceConfig,
       "dataset" -> JString(formatDatasetId(name)),
       "schema" -> jsonifySchema(schema))
 
+  // Repeatedly tries to get a new thread by bumping numThreads, using
+  // compareAndSet to make sure our value is valid
+  @annotation.tailrec
+  private def tryGetMutationThread(numTries: Int = 100): Boolean = {
```

Comment:
True.   I'll change this to accept a duration, and to make it configurable.   Maybe 5 minutes default?

As for the max Jetty thread pool, we should do that too, but possibly another PR.